api_server/Pipfile

```diff
@@ -0,0 +1,16 @@
+[[source]]
+name = "pypi"
+url = "https://pypi.org/simple"
+verify_ssl = true
+
+[dev-packages]
+
+[packages]
+flask = "*"
+numpy = "*"
+gunicorn = "*"
+flask-cors = "*"
+pandas = "*"
+
+[requires]
+python_version = "3.7"
```

Comment:
Might want to think about making this not so explicit in terms of version. I know I've had issues in the past with this. You should just be able to use `3` and it be okay I think?  Or is there syntax where you can go say `>=3.7` ? I know you can do that in package.json for version numbers.